joomla/joomla-cms
# Changes Session Handler text from "None" to "PHP"

#### Description

Language change to 'None' Session handler that is not correct. Is not None, is PHP session handler.
#### How to test

Go to System -> Global configurarion, System separator, Session Settings block.
In 'Session handler' should be at least 'Database' and 'PHP'. Check tooltip too.
